src/group/stabchain/builder/mod.rs

```diff
@@ -28,7 +28,7 @@ where
 
 /// A strategy is a lightweight struct that allows to
 /// (hopefully at compile time plz compiler) select which builder to use
-pub trait BuilderStrategy<P> {
+pub trait BuilderStrategy<P>: Clone {
```

Comment:
Do we need this bound?

Reply:
I added it in because strategies ended up being cloned a lot in the benchmarking and testing, so thought it may be good to specify this requirement. Currently all builders were implementing this anyway.

Reply:
I think it's better to remove it and leave it as generic as possible, if we are not cloning it into the generic code and only in the implementations I think that it will be better to remove it.